docs/source/tutorial/registration.md

```diff
@@ -74,7 +78,9 @@ The common loss function is Dice loss, Jacard and average cross-entropy over all
 
 ### Deformation loss
 
-This type of loss measures the deformation to regularize the transformation.
+This type of loss measures the amount of deformation and penalises severe deformations.
+Penalising severe deformations helps to regularise the transformation between fixed and
```

Comment:
penalize or penalise lol... fine I will let it go as most of us use British english